Dockerfile.okd

```diff
@@ -0,0 +1,65 @@
+# Please ensure this file is kept in sync with Dockerfile.ocp
+FROM fedora:33 AS builder
```

Comment:
Is there a reason we have to use fedora for the builder, vs just using centos8 for both the builder and image base?

Reply:
We need the `shim` RPM from fedora repos, and I figured it would be easier to use fedora as it's got the repos already configured correctly, so it can just be installed with dnf.
If this is a blocker we could of course use a centos image and pull the RPM manually from Fedora repos

Reply:
No it's fine, I just wondered if we needed to pull both base images vs just using the centos one in both places :)

Reply:
Yeah, the UEFI images signatures must match here and in the images.